src/slack/views/registerTeamSubmitted.ts

```diff
@@ -112,23 +115,16 @@ Team Members: ${formattedTeamMembers.join(', ')}
       users: registeringUser,
     })) as DmOpenResult;
 
-    const formattedTeamMembers = allTeamMembers.map((member) => `<@${member}>`);
     await app.client.chat.postMessage({
       token: context.botToken,
       channel: dm.channel.id,
-      text: `:warning: Something went wrong while registering your team... come chat with our team for help.
-
-To help with resubmitting, here's the info you tried to submit:
-Team Name: ${teamName}
-TableNumber: ${tableNumber}
-Project Description: ${projectDescription}
-Team Members: ${formattedTeamMembers.join(', ')}
-
-Here's what went wrong, it may be helpful (but probably not):
-\`\`\`
-${JSON.stringify(err, null, 2)}
```

Comment:
I am not sure if we are covering this line when passing the string from dictionary.

Reply:
I’m not following your question

Reply:
I couldnt make the team registration fail. I am not sure if this JSON is available to user when the registration fails.

Reply:
There was a problem me not being able to fail the team registration. I solved it.